README.md

```diff
@@ -51,7 +51,7 @@ If you need BPMN examples, you can use resources from
 | Yes | Yes | Yes | Yes |
```

Comment:
Is it possible to replace 'Yes' by 'check' icon like done in the HTML doc ?

Reply:
yes we can use emoji, I will do that